### DISCUSSION TITLE
common_names in certificates

### DISCUSSION THREAD

**Discussion by ilvpozzd** [siderolabs/talos#5352] [Q&A] [answered]
I generate certificates without talosctl (using openssl e.g.) Does this certificates must have "common_name" attribute with init node/all nodes IPs/Domain Names?

**Comment by smira** [comment, marked as answer]
Which certificate? Server? Client?

You can look into generated certificates to get an idea on what should be there.

Or, better, use Talos Go client to do the custom stuff you're looking for
